systemtest/loadgen/eventhandler.go

```diff
@@ -35,13 +35,13 @@ var events embed.FS
 
 // NewEventHandler creates a eventhandler which loads the files matching the
 // passed regex.
-func NewEventHandler(p, url, token string, l *rate.Limiter) (*eventhandler.Handler, error) {
+func NewEventHandler(p, url, token string, l *rate.Limiter, apiKey string) (*eventhandler.Handler, error) {
```

Comment:
I wanted to put `apiKey` close to `token` param but it could break the existing usage so I put it as the last argument just to be safe, but I'm happy to reorganise the implementations with the new order after putting the `apiKey` next to `token`. 

In my previous experience, I prefer to pass params as a struct (object to be precise as it was javascript :p) when there are many arguments for a function so that the order of the params doesn't matter.

Reply:
> In my previous experience, I prefer to pass params as a struct (object to be precise as it was javascript :p) when there are many arguments for a function so that the order of the params doesn't matter.

I would be happy to introduce a config/params struct for this function.